tasks.py

```diff
@@ -634,20 +635,21 @@ def docs(
     elif version:
         docs_builder = DocsBuilder(ctx, docusaurus_dir)
         docs_builder.create_version(version=parse_version(version))
-    else:  # noqa: PLR5501
-        if start:
-            ctx.run(" ".join(["yarn start"]), echo=True)
+    elif start:
+        ctx.run(" ".join(["yarn start"]), echo=True)
+    elif clear:
```

Comment:
This is the new condition.

The rest of the change here is pulling the start and embedded else branches to the top level to remove PLR5501:
```
else:
  if start:
     ...
  else:
```